CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Changes
 
+  * [#774](https://github.com/toptal/chewy/pull/774): Remove CircleCI. Setup CI on GitHub Actions ([@Vitalina-Vakulchyk][])
+
```

Comment:
This is not something user-visible, we don't need a CHANGELOG entry for this PR.